addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/OpenApiSpider.java

```diff
@@ -83,7 +84,7 @@ public boolean canParseResource(HttpMessage message, String path, boolean wasAlr
         } catch (Exception e) {
             return false;
         }
-        log.debug("Cant parse " + message.getRequestHeader().getURI());
+        log.debug("Can't parse {}", message.getRequestHeader().getURI().toString());
```

Comment:
The `toString()` should not be added, avoids converting to string sooner than needed.

Reply:
That's my fault (I think), I was adding toString in the other logging PRs in order to make it clear what was going on, but if just using `message.getRequestHeader().getURI()` and letting the JRE handle it is preferred I can re-tweak all those as well.

Reply:
Hmm, I missed those then. Yes, better not create the string if it's not needed.